inventory/packages/apt_deb.go

```diff
@@ -112,17 +122,17 @@ func AptUpdates() ([]PkgInfo, error) {
 		if pkg[0] != "Inst" {
 			continue
 		}
-		if len(pkg) == 5 {
+		if strings.HasPrefix(pkg[2], "(") {
 			// We don't want to record new installs.
 			// Inst firmware-linux-free (3.4 Debian:9.9/stable [all])
 			continue
 		}
-		if len(pkg) != 6 {
+		if strings.HasPrefix(pkg[2], "[") && strings.HasPrefix(pkg[3], "(") && strings.HasSuffix(pkg[len(pkg)-1], "]") {
```

Comment:
When doing this type of check, it may be helpful to include an example of what this is looking for in the comments. That way if there are false positives detected in the future, cases that are similar but need to be handled differently, it'll be clear what this check is looking for. (Like what you have for the last check. But maybe add "eg." to make it clearer.)

Reply:
Nice catch, this actually doesn't do what I want (I had caught it in my tests but those aren't in this PR)